src/css/tabs/firmware_flasher.less

```
@@ -123,7 +123,12 @@
 			}
 		}
 	}
-	.release_info {
+    .build_configuration {
+        .select2-selection__choice {
+            color: #3f4241;
```

Remove the 30px gap:

```suggestion
            margin-left: auto;
            color: #3f4241;
```